tools/README.md

```diff
@@ -430,6 +430,9 @@ Failed jobs:
 +-----+------+--------+---------------------------------------------------------------------------------------------------+
 ```
 
+### Metrics Definitions
+All the metrics definition can be found in [executor task metrics doc](https://spark.apache.org/docs/latest/monitoring.html#executor-task-metrics) / [executor metrics odc](https://spark.apache.org/docs/latest/monitoring.html#executor-metrics) or [spark webUI doc](https://spark.apache.org/docs/latest/web-ui.html#content).
```

Comment:
misspelled doc after executor metrics, Please capitalize spark in "spark webUI

Reply:
Done.